airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -114,6 +115,15 @@ public AirbyteConnectionStatus check(final JsonNode config) {
     }
   }
 
+  /**
+   * Specific Databases may have additional checks unique to them which they need to perform, override
+   * this method to add additional checks.
+   *
+   * @param database the database to run checks against
+   * @throws Exception
+   */
+  protected void destinationSpecificTableOperations(final JdbcDatabase database) throws Exception {}
```

Comment:
what do you think of having redshift override the `check` method instead of adding a new overridable method?
```java
class RedshiftDestination {
  @Override
  AirbyteConnectionStatus check(config) {
    status = super.check(config);
    if (status != SUCCESS) { return status }
    if (!svvTableInfoAccess(config)) { return FAILED }
    return SUCCESS;
  }
}
```